aws/s3.tf

```diff
@@ -123,3 +123,94 @@ module "s3_bucket" {
 
   tags = local.tags
 }
+# aws S3 backup //////////////////////
+resource "aws_backup_vault" "datagrok_public_vault" {
```

Comment:
The resource name will be the same for us and for clients. It is not right to add our bucket name to the name of the resource. The name should be general, like for other resources. For example, `s3_backup_plan`